hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -760,8 +767,26 @@ public ScmInfo getScmInfo() throws IOException {
         .setScmId(resp.getScmId())
         .setRatisPeerRoles(resp.getPeerRolesList());
 
-    return builder.build();
+    // By default, we assume that SCM Ratis is not enabled.
+    builder.setScmRatisEnabled(Boolean.FALSE);
```

Comment:
Parameter is `boolean`, but we're passing `Boolean`, why?

Reply:
I should use a `boolean` variable, but I wanted to represent it with a constant and ended up overlooking the difference between `Boolean` and `boolean`.  I should have used `true` and `false` directly.

Reply:
This is still `FALSE`.  In fact, we can remove it since builder starts out with `false`.

```suggestion
```